jquery.timeago.js

```diff
@@ -113,11 +113,11 @@
 
       var separator = $l.wordSeparator || "";
       if ($l.wordSeparator === undefined) { separator = " "; }
-      return $.trim([prefix, words, suffix].join(separator));
+      return String.prototype.trim.call([prefix, words, suffix].join(separator));
```

Comment:
[QUESTION/CONCERN] Why isn't this:

```js
      return [prefix, words, suffix].join(separator).trim();
```

Reply:
It can but i thought a less impactful change was calling `call` to 1-1 replace `$.trim`.  The other way inverses the code.  I have no problem doing it but a lot of people were recommending doing it this way as a search and replace.  But its a matter of preference I can change it no problem